libraries/botbuilder-ai/tests/qnaMaker.test.js

```diff
@@ -18,29 +20,45 @@ class TestContext extends TurnContext {
     }
 }
 
-xdescribe('QnAMaker', function () {
+describe('QnAMaker', function () {
     this.timeout(10000);
```

Comment:
Juan, the line timeout can be removed and the tests will pass, but may I ask why remove it?

Reply:
This shouldn't be here in the first place. We found some issues with things like this in the build server. It is better to define timeout from outside the test code, it can be changed, tweaked via config files. This ways, you have to edit code to do it.
BTW, it is not related with the task you are doing right now, but it is better to fix it ASAP.

Reply:
Ok, no problem. I am going to make a new commit based on your feedback.